content/components/select/disabled.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: component/code-demo
+iframe: examples/example-disabled
+iframeFullwidth: true
+code:
+  - HTML: |
+      <!--
+      Valid:    <label class="au-select" disabled>
+      Invalid:  <label class="au-select" disabled>
+      -->
```

Comment:
Lets follow what we have done here for the above few lines

https://designsystem.gov.au/components/text-inputs/#disabled-text-inputs